modules/consuming-huge-pages-resource-using-the-downward-api.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/what-huge-pages-do-and-how-they-are-consumed-by-apps.adoc
+
+:file-name: hugepages-volume-pod.yaml
+
+[id="consuming-huge-pages-resource-using-the-downward-api_{context}"]
+= Consuming huge pages resources using the Downward API
+
+You can use the Downward API to inject information about the huge pages resources that are consumed by a container.
```

Comment:
Not sure, but you have "huge pages resources". Should it be "huge page resources"?

Reply:
There are numerous instances of "huge pages" so I'm inclined to have this one match.